Cogs/ServerManagement.py

```diff
@@ -206,7 +207,8 @@ async def rolemenu(self, ctx):
 
                 # If role, make button style gray. If URL, make style URL
                 if not validators.url(role_link):
-                    buttons.append(Button(style=ButtonStyle.gray, label=text, emoji=emoji))
+                    # buttons.append(Button(style=ButtonStyle.gray, label=text, emoji=emoji))
+                    buttons.append(Button(style=ButtonStyle.gray, label=text, emoji=await get_emoji_named(ctx.guild, emoji)))
```

Comment:
Maybe add some sort of check to see if the server has an emoji with the name, and if not fall back to the default emojis?